packages/MJServer/src/resolvers/AskSkipResolver.ts

```diff
@@ -18,7 +18,7 @@ import { registerEnumType } from "type-graphql";
 import { MJGlobal, CopyScalarsAndArrays } from '@memberjunction/global';
 import { sendPostRequest } from '../util';
 import { GetAIAPIKey } from '@memberjunction/ai';
-import { PrimaryKeyValueInputType } from './MergeRecordsResolver';
+import { KeyValuePairInputType } from './MergeRecordsResolver';
```

Comment:
@JS-BC  again --- why? We should have CompositeKeyInputType and use that CLEANLY everywhere so if in the future we have MORE information in the CK class things still work. This will fail in the future.